apps/webapp/src/components/funnel-icon-button.tsx

```diff
@@ -0,0 +1,86 @@
+import { Box, Popover, Portal } from '@chakra-ui/react';
+import EmojiPicker from 'emoji-picker-react';
+import { useState } from 'react';
+import { TbChartFunnel } from 'react-icons/tb';
+import type { CardIconProps } from './card-icon';
+import { CardIcon } from './card-icon';
+import { MenuContent, MenuContextTrigger, MenuItem, MenuRoot } from './ui/menu';
+
+interface Props extends CardIconProps {
+  icon?: string | null;
+  onIconChange?: (icon: string) => void;
```

Comment:
I'd prefer only an onIconChange callback in this case with icon: string | null and calling it with null in case of removing